presentations.yaml

```diff
@@ -93,3 +93,24 @@
     - Keynote
     - Project Maintainer
     - Ambassador
+- name: "Beyond the Magic: Disaster Recovery Strategies in Crossplane and GitOps"
+  description: |
+    In Crossplane, deploying resources using GitOps (ArgoCD) is straightforward, but disaster recovery is a different ballgame. When calamity hits, you can't just spin up a new Kubernetes cluster, point it to your Git repository, and expect a 'magical' restoration. This talk will demystify the notion that disaster recovery is 'magic' by exploring various strategies tailored for different levels of disasters using Velero.
+  slides: https://www.slideshare.net/CarlosSantana1/navigating-disaster-recovery-in-kubernetes-and-cncf-crossplane
+  video: https://youtu.be/6qYFmAy1cFA
+  date: 2023-05-08
+  presenters:
+    - name: Carlos Santana
+      github: csantanapr
+  event:
+    name: GitOpsCon NA 2023
+    url: https://sched.co/1Jp8D
+  language: EN
+  projects:
+    - Kubernetes
+    - Crossplane
+    - Velero
```

Comment:
Velero isn't a CNCF project is it? Have we decided that it's ok to list any software projects in this field even if they are not CNCF projects?

Reply:
I will send PR to not create a precedence, and for now only accept cncf projects in this field @castrojo

Reply:
Ok, thanks.